src/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { SwitchWrapper, SwitchWrapperProps } from './SwitchWrapper';
+
+export default {
+  title: 'Components/Switch',
+  argTypes: { onChange: { action: 'changed' } },
+  component: SwitchWrapper,
+} as Meta;
+
+// TODO this one does not work currently
```

Comment:
Are you leaving this one for another PR? Bit concerned that it may not work when it gets into the app either and block us from updating. Or is it just the story that is not working?

Reply:
It should be the story that does not work, I havent changed the Switch code. We will be able to make the stories work but needs more time and dedicated attention onto it